### PR TITLE
Bug 1854659: Handle users' privacy aspects

### PR DESCRIPTION
**Pull request by abhi-agg**
Fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1854659

1. Added a footer to link [Mozilla's privacy policy](https://www.mozilla.org/en-US/privacy/websites/) (taking inspiration from https://telemetry.mozilla.org/ which is a public website),
2. In the same footer, added one more line explaining how to [disable data collection](https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature)
3. Updated README to explain how to [disable data collection](https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature)

### CODE REVIEW

**Review by rosahbruno — APPROVED** [review]
Style-wise I think we need to do a few things:
1. Make sure this is always on the bottom of the screen. If you look at a screen like http://localhost:3000/pings/ppop-iECO4, you can see the footer half way up the page. We need to add some sort of positioning CSS here to keep it always at the bottom.
2. Can we update the styling of this a bit to match with our theming? The component is the same color for both themes and stands out a bit compared to the rest of the elements. I think we could make it the same color as the rest of the page just give it a border and make sure the text color switches with it.

I know you are supposed to be on PTO either today or tomorrow. Since this is the last part of the stuff you've been working on, if you need to land this go ahead and I can update the styles later. I don't want to hold you or this up for styling changes.

File: README.md [hunk 1]
```diff
@@ -72,4 +72,7 @@ Pings are retained for a limited time. Every day, a [function](functions/garbage
 Views are leveraging Firestore's realtime query updates.
 
 #### Authentication
-Simple Google Signin-based authentication is inplace. It allows to log in with any Google account, although only those under `mozilla.com` domain are [allowed](firestore.rules) to fetch data from Firestore.
\ No newline at end of file
+Simple Google Signin-based authentication is inplace. It allows to log in with any Google account, although only those under `mozilla.com` domain are [allowed](firestore.rules) to fetch data from Firestore.
+
+#### Data Collection
+This project uses [Glean.js SDK](https://github.com/mozilla/glean.js) for telemetry data collection. If you want to disable data collection, please [turn on browser's Do Not Track](https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature) feature.
```
Comment (Dexterp37) [inline README.md:78]:
```suggestion
This project uses [Glean.js SDK](https://github.com/mozilla/glean.js) for data collection. If you want to disable data collection, please [turn on browser's Do Not Track](https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature) feature.
```

Comment (abhi-agg) [reply README.md]: Done.

**Comment by rosahbruno** [comment]
I am going to merge this PR. I've logged a bug for handling the UI follow ups that I called out above.

https://bugzilla.mozilla.org/show_bug.cgi?id=1857007

**Pull request merged**